crates/leon/src/lib.rs

```diff
@@ -121,6 +121,18 @@
 //! assert_eq!(template.render(&values).unwrap().as_str(), "hello pontifex");
 //! ```
 //!
+//! # Parse template at compile time
+//!
+//! [`leon-macros`] provides proc-macros [`leon_macros::const_parse_template!`]
+//! to parse the template at compile time.
+//!
+//! NOTE that in order to use that it, you would have to add leon as a dependency
+//! to your crate first.
+//!
+//! If proc-macro is way too costy for you, have a try at [`leon::template`],
+//! a light-weight alternative that requires you to split items into text and
+//! keys yourself but does not bring in proc-macro2/syn/quote.
```

Comment:
```suggestion
//! # Compile-time parsing
//!
//! You can either use [`leon-macros`]'s [`template!`](leon_macros::template!), a proc-macro, with the exact same syntax as the normal parser, or this crate's [`template!`] rules-macro, which requires a slightly different syntax but doesn't bring in additional dependencies. In either case, the leon library is required as a runtime dependency.
```

I like that name and presentation better. Also would be good to keep some tests using the rules-macro to ensure it remains covered.

Reply:
Thanks!

I've renamed it to `leon_macros::template!` and updated the doc according to your suggestion, though I cannot apply it directly due to the infra links doesn't work.

I've also fixed other broken infra links.